integration-tests/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
```

Comment:
Could we remove a layer of directories and just have everything in `src/tests` rather than `src/tests/integration`?

(I think it's a given that any tests in the separate `tests` folder are already integration tests anyway, really.